bedrock/firefox/templates/firefox/nightly_whatsnew.html

```diff
@@ -2,13 +2,12 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "/firefox/base/base-pebbles.html" %}
-{% add_lang_files "firstrun" "mobile" %}
```

Comment:
Note for reviewer: as far as I can tell (?) these lang files don't exist, and removing them does not impact page translations.